### PR TITLE
Updates to support for 3.7.0

### PR DESCRIPTION
**Pull request by rozza**

### CODE REVIEW

File: docs/reference/content/upgrade.md [hunk 1]
```diff
@@ -8,10 +8,21 @@ title = "Upgrading"
...
 
+### 2.3.0
+
+#### MongoClientSettings
+
+The Mongo Java Driver 3.7.0 introduces a new `com.mongodb.MongoClientSettings` class to unify the settings across the sync and async drivers.
+The legacy `com.mongodb.async.client.MongoClientSettings` has been deprecated as such the `MongoClientSettings` alias now points to the
+supported class and the `MongoClientSettings.builder()` helper points to the supported builder.
+
+The legacy settings are still supported and can imported from `com.mongodb.async.client.MongoClientSettings`. It is only required if you
```
Comment (jyemin) [inline docs/reference/content/upgrade.md]: "can be imported from"

File: docs/reference/content/upgrade.md [hunk 2]
```diff
@@ -8,10 +8,21 @@ title = "Upgrading"
...
+need multiple credentials or custom `heartbeatSocketSettings` both of which have been deprecated.
```
Comment (jyemin) [inline docs/reference/content/upgrade.md]: comma before "both"

File: docs/reference/content/upgrade.md [hunk 3]
```diff
@@ -8,10 +8,21 @@ title = "Upgrading"
 
 ## Upgrade
 
+### 2.3.0
+
+#### MongoClientSettings
+
+The Mongo Java Driver 3.7.0 introduces a new `com.mongodb.MongoClientSettings` class to unify the settings across the sync and async drivers.
+The legacy `com.mongodb.async.client.MongoClientSettings` has been deprecated as such the `MongoClientSettings` alias now points to the
```
Comment (jyemin) [inline docs/reference/content/upgrade.md]: Period after "deprecated"

File: docs/reference/content/upgrade.md [hunk 4]
```diff
@@ -8,10 +8,21 @@ title = "Upgrading"
...
+supported class and the `MongoClientSettings.builder()` helper points to the supported builder.
+
+The legacy settings are still supported and can imported from `com.mongodb.async.client.MongoClientSettings`. It is only required if you
+need multiple credentials or custom `heartbeatSocketSettings` both of which have been deprecated.
+
 ### 2.0.0
 
 
-#### MongoCollection mehtod default to collection type fix.
+#### MongoCollection method default to collection type fix.
```
Comment (jyemin) [inline docs/reference/content/upgrade.md:25]: Good eyes

File: project/MongoScalaBuild.scala [hunk 5]
```diff
@@ -32,7 +32,7 @@ object MongoScalaBuild extends Build {
   import Dependencies._
   import Resolvers._
 
-  val baseVersion = "2.2.1"
+  val baseVersion = "3.0.0"
```
Comment (jyemin) [inline project/MongoScalaBuild.scala]: What does this number refer to?

**Review by jyemin — CHANGES_REQUESTED** [review]
Looks great.

Just a few grammar changes, and a question.

File: project/MongoScalaBuild.scala (hunk 5 above)
Comment (rozza) [reply project/MongoScalaBuild.scala]: Good catch thats the base version - but we no longer need to do a major release.

File: docs/reference/content/upgrade.md (hunk 1 above)
Comment (rozza) [reply docs/reference/content/upgrade.md]: Done

File: docs/reference/content/upgrade.md (hunk 2 above)
Comment (rozza) [reply docs/reference/content/upgrade.md]: Done

File: docs/reference/content/upgrade.md (hunk 3 above)
Comment (rozza) [reply docs/reference/content/upgrade.md]: Done

**Comment by rozza** [comment]
Updated

**Review by jyemin — APPROVED** [review]

**Comment by jyemin** [comment]
LGTM assuming the failing Travis checks are a false alarm

**Comment by jyemin** [comment]
LGTM again

**Pull request closed without merging**